tools/shmsnoop.py

```diff
@@ -238,6 +238,28 @@ def sys_name(sys):
   { 'name' : 'SHM_EXEC',   'value' : 0o100000 },
 ]
 
+shmctl_cmds = [
+    ('IPC_STAT', 2),
+    ('IPC_SET', 1),
+    ('IPC_RMID', 0),
+    ('IPC_INFO', 3),
+    ('SHM_INFO', 14),
+    ('SHM_STAT', 13),
+    ('SHM_STAT_ANY', 15),
+    ('SHM_LOCK', 11),
+    ('SHM_UNLOCK', 12),
```

Comment:
Could you sort the above table based on command number?

Reply:
Thanks, Just finished. please review.